lib/openstudio-standards/prototypes/common/objects/Prototype.hvac_systems.rb

```diff
@@ -749,6 +749,7 @@ def model_add_hp_loop(model,
       when 'FluidCoolerTwoSpeed'
         cooling_equipment = OpenStudio::Model::FluidCoolerTwoSpeed.new(model)
         cooling_equipment.setName("#{heat_pump_water_loop.name} FluidCoolerTwoSpeed")
+        fluid_cooler_apply_minimum_power_per_flow(cooling_equipment)
```

Comment:
Applying efficiency standards should not be a part of this part of code.  It should instead be called here:
https://github.com/NREL/openstudio-standards/blob/large_office_fluid_cooler/lib/openstudio-standards/standards/Standards.Model.rb#L1545-L1549